--- #Publish all joint states ----------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 500

#Position Controller
position_controller:
  type: "position_controllers/JointGroupPositionController"
  joints:
    - arm_joint_1
    - arm_joint_2
    - arm_joint_3
    - arm_joint_4
    - arm_joint_5
    - arm_joint_6
    - arm_joint_7
